manifests/plugin/abrt.pp

```diff
@@ -33,7 +35,8 @@
 #
 class foreman_proxy::plugin::abrt (
   $enabled                 = $::foreman_proxy::plugin::abrt::params::enabled,
-  $version                 = undef,
```

Comment:
How about everything or nothing? That means there's always one place to look for the defaults.